WebUI: Make m-VO compatible with WebUI, Closes #3885

patrick-austin opened the pull request:
Make m-VO compatible with WebUI
------------------

VO is now either determined based on identity (with account name narrowing the options if provided) at login. If multiple VOs are possible for the identity given, then the user stays on the login screen and has to manually choose their VO from a drop down list.

This VO is then set as a cookie and used in the headers so information displayed only applies to the user's VO.

Should be no change for single-VO operation.

patrick-austin committed:
WebUI: Make m-VO compatible with WebUI, Closes #3885

tbeerman commented on lib/rucio/web/ui/common/utils.py:
```diff
@@ -73,6 +75,7 @@
     except:
         AUTH_ISSUERS = []
 
+MULTI_VO = config_get('common', 'multi_vo', default=False)
```
please use `config_get_bool` here. Otherwise multi_vo will always be enabled as soon it's added to the config file, even for `multi_vo = False`

patrick-austin replied:
Done.

tbeerman commented on lib/rucio/web/ui/main.py:
```diff
@@ -83,6 +84,7 @@
 
 )
 
+MULTI_VO = config_get('common', 'multi_vo', default=False)
```
see above

patrick-austin replied:
And done.

tbeerman reviewed (changes requested):
Looks good to me. It couldn't really test the multi_vo features, since I don't have a multi_vo test env, but at least it doesn't break anything in my single vo test env.

patrick-austin commented:
Updating from `config_get` to `config_get_bool` broke all the tests because I didn't set `raise_exception=False`. Hopefully the tests should pass now I've changed that.

I also added a missing default argument for `ui_vo` in the `oidc_auth` function.
